### DISCUSSION TITLE
Standalone build

### DISCUSSION THREAD

**Discussion by fabis94** [apollographql/apollo-client-devtools#1046] [Ideas]
Similar to how Vue dev tools have a standalone electron app: https://devtools.vuejs.org/guide/installation.html#standalone

This is useful when debugging Apollo apps running in restrictive environments like CefSharp WebViews where you can't install any extra extensions like you could in a normal browser.

**Comment by alessbell** [comment]
We do release the devtools as the standalone package `apollo-client-devtools` - I'll look into whether this is already possible and update this discussion in a bit!
